modules/atf_logger.lua

```diff
@@ -1,10 +1,37 @@
 local json = require("json")
 local Logger = {}
 Logger.mobile_log_format = "%s(%s) [version: %s, frameType: %s, encryption: %s, serviceType: %s, frameInfo: %s, messageId: %s] : %s \n"
-Logger.hmi_log_format = "%s(%s) : %s \n"
+Logger.hmi_log_format = "%s[%s] : %s \n"
+
+
+function capture_cmd_output(cmd, is_raw)
+  local f = assert(io.popen(cmd, 'r'))
+  local s = assert(f:read('*a'))
+  f:close()
+  if is_raw then return s end
+  s = string.gsub(s, '^%s+', '')
+  s = string.gsub(s, '%s+$', '')
+  s = string.gsub(s, '[\n\r]+', ' ')
+  return s
+end
+
+function split(inputstr, sep)
+  if sep == nil then
```

Comment:
@LevchenkoS I would suggest to use style like here: [Lua Style Guide](http://lua-users.org/wiki/LuaStyleGuide) . Paragraph: "Lua Idioms"